postgres-appliance/scripts/configure_spilo.py

```diff
@@ -389,7 +389,7 @@ def get_instance_metadata(provider):
         mapping = {}  # Disable multi-url fetch
         url = 'http://169.254.169.254/openstack/latest/meta_data.json'
         openstack_metadata = requests.get(url, timeout=5).json()
-        metadata['zone'] = openstack_metadata.availability_zone
```

Comment:
there is no dealing at this point with errors?

Reply:
Yes, the old implementation just crashed. If the current MR has an issue the same would happen.